docs/administration/audit-logs.md

```diff
@@ -0,0 +1,41 @@
+# Audit Logs
+
+Audit logs are a record of all user actions performed on the platform across experiments, feature flags, metrics, and facts.
+
+Audit logs are written to your warehouse in the `merged_eppo.eppo_entity_change_logs` table. Audit logs are updated every 24 hours.
```

Comment:
If I understand correctly, I think `merged_eppo.` should be omitted, since that varies depending on the customer's warehouse setup (database name and whatnot).

Reply:
we actually synchronize every hour, not every 24 hours